include/micm/solver/cuda_lu_decomposition.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+#include<micm/solver/lu_decomposition.hpp>
+#include<micm/util/cuda_param.hpp>
+#include <stdexcept>
+#ifdef USE_CUDA
+#include <micm/solver/cuda_lu_decomposition.cuh>
+#endif 
+
+#ifdef USE_CUDA
+namespace micm{
+    class CUDALuDecomposition: public LuDecomposition{
```

Comment:
```suggestion
    class CudaLuDecomposition: public LuDecomposition{
```
please rename for consistency with `CudaProcessSet` and to follow style of capitalizing only first character of acronyms in class/stuct/function names

Reply:
Done.